internal/policy/verify.go

```diff
@@ -375,6 +375,11 @@ func verifyEntry(ctx context.Context, repo *git.Repository, policy *State, entry
 			gitNamespaceVerified = true
 			break
 		}
+		if errors.Is(err, gitinterface.ErrUnknownSigningMethod) {
+			// We encounter this for key types that can be used for metadata
```

Comment:
It's not immediate clear what metadata means here - can we add more details?

Reply:
fixed!